governance-charter.md

```diff
@@ -232,3 +232,44 @@ majority vote of the governance committee members, but closed meetings do not
 count towards the required meeting frequency. The governance committee will
 inform the community if a particular meeting is closed, as well as the general
 reasons for this, prior to the meeting taking place.
+
+## Community Manager Role
+
+In order to facilitate end-user and contributor experience programs that cut
+across multiple SIGs in the OpenTelemetry project, the governance committee
+may appoint Community Managers (CMs) to act as liaisons, facilitators, and 
+organizers for such programs. The goal of this role is to ensure the health of
+the OpenTelemetry end-user and contributor community.
+
+### Community Manager Responsibilities
+
+- Act as stewards for the OpenTelemetry end-user and contributor community by
+  organizing events and coordinating programs.
+- Manage the OpenTelemetry social media and blog presence.
+- Work with the CNCF and member organizations on cross-functional projects as
+  needed.
+- Track and report on OpenTelemetry end-user and contributor community health
+  for the GC.
+- Grow the OpenTelemetry end-user and contributor community by advocating for
+  programs and initiatives that will attract new users or aid contributor
+  experience.
+  
+### Community Manager Eligibility, Term, and Selection
+
+CMs shall be nominated and approved by the GC for one-year terms on an as-needed 
+basis, by simple majority vote. The GC shall re-confirm CMs on an annual basis.
+A CM that wishes to step down from their role shall notify the GC at least 30 days
+before the next GC meeting.
+
+The GC may appoint individuals who are not part of the OpenTelemetry project. 
+
+The GC may dismiss a CM for violations of the project Code of Conduct or a 
+failure to uphold the responsibilities of the role.
+
+### Community Manager Rights and Permissions
+
+A CM may act as an extension of the GC in pursuing its responsibilities. They
```

Comment:
Can we add a sentence about how the GC should hold CMs accountable for decisions made "as an extension of the GC", and that CMs should check in before making potentially controversial decisions in this capacity?

Reply:
Added a new statement here clarifying this.